src/tokenizer/tokenizer.py

```diff
@@ -2223,14 +2238,14 @@ def lookup(abbrev: str) -> Optional[List[BIN_Tuple]]:
                 ):
                     # Ordinal, i.e. whole number or Roman numeral followed by period:
                     # convert to an ordinal token
-                    follow_token = token_stream[0]
-                    if follow_token and not (
-                        follow_token.kind in TOK.END
-                        or follow_token.punctuation in {"„", '"'}
+                    ord_token: Optional[Tok] = token_stream[0]
```

Comment:
For some reason, using the name follow_token caused typing errors since it had been defined above differently. This only happened a few times (and didn't seem to follow rhyme nor reason), which might explain why it passed inspection when it was originally added and merged.